use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+        * [Install necessary bundles for Clear Linux and disable kernel updates](#install-necessary-bundles-for-clear-linux-and-disable-kernel-updates)
+        * [Replace Clear Linux kernel with LTS kernel](#replace-clear-linux-kernel-with-lts-kernel)
+        * [Allow insecure modules to load to host](#allow-insecure-modules-to-load-to-host)
+        * [Download Intel QAT drivers](#download-intel-qat-drivers)
+        * [Compile Intel QAT drivers against host kernel](#compile-intel-qat-drivers-against-host-kernel)
+        * [Copy Intel QAT configuration files and enable Virtual Functions](#copy-intel-qat-configuration-files-and-enable-virtual-functions)
+        * [Expose and Bind Intel QAT virtual functions to VFIO-PCI (Every reboot)](#expose-and-bind-intel-qat-virtual-functions-to-vfio-pci-every-reboot)
+        * [Check Intel QAT virtual functions are enabled](#check-intel-qat-virtual-functions-are-enabled)
+    * [Prepare Kata Containers](#prepare-kata-containers)
+        * [Download Kata kernel Source](#download-kata-kernel-source)
+        * [Build Kata kernel](#build-kata-kernel)
+        * [Copy Kata kernel](#copy-kata-kernel)
+        * [Prepare Kata root filesystem](#prepare-kata-root-filesystem)
+        * [Compile Intel QAT drivers for Kata Containers kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
+        * [Copy Kata rootfs](#copy-kata-rootfs)
+        * [Update Kata configuration to point to custom kernel and rootfs](#update-kata-configuration-to-point-to-custom-kernel-and-rootfs)
+    * [Verify Intel QAT works in a Docker Kata Containers container](#verify-intel-qat-works-in-a-docker-kata-containers-container)
+    * [Build OpenSSL Intel QAT engine container](#build-openssl-intel-qat-engine-container)
+        * [Test Intel QAT in Docker](#test-intel-qat-in-docker)
+        * [Troubleshooting](#troubleshooting)
+    * [Optional Scripts](#optional-scripts)
+        * [Verify Intel QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
+
+
+# Introduction
+
+Intel QuickAssist Technology (Intel QAT) provides hardware acceleration 
+for security (cryptography) and compression. These instructions cover the 
+steps for [Clear Linux](https://clearlinux.org) but can be adapted to any 
+Linux distribution. Your distribution may already have the Intel QAT 
+drivers, but it is likely they do not contain the necessary user space 
+components. These instructions guide the user on how to download the kernel 
+sources, compile kernel driver modules against those sources, and load them 
+onto the host as well as preparing a specially built Kata Containers kernel 
+and custom Kata Containers rootfs.  
+
+## Helpful Links before starting
+
+[Intel QAT Engine](https://github.com/intel/QAT_Engine)
+
+[Intel QuickAssist Technology at 01.org](https://01.org/intel-quickassist-technology)
```

Comment:
Hi @eadamsintel - there is just *one* final change to make to this doc before we can land it...

Travis is failing on the word "org". This is because Travis uses Ubuntu Xenial (16.04) and the hunspell dictionaries in that distro version don't understand "org" as a word (but never versions of those dictionaries do). On Xenial:

```sh
$ echo "organ" | hunspell -d en_US,en_GB -l  # No output means "accepted"
$ echo "org" | hunspell -d en_US,en_GB -l    # Gaah!
org
```

The fix is simple - just put backticks around `01.org` when it appears in the *name* of a link (three occurrences to change):

```diff
diff --git a/use-cases/using-Intel-QAT-and-kata.md b/use-cases/using-Intel-QAT-and-kata.md
index dc39072..8ff8186 100644
--- a/use-cases/using-Intel-QAT-and-kata.md
+++ b/use-cases/using-Intel-QAT-and-kata.md
@@ -48,7 +48,7 @@ and custom Kata Containers rootfs.
 
 [Intel QAT Engine](https://github.com/intel/QAT_Engine)
 
-[Intel QuickAssist Technology at 01.org](https://01.org/intel-quickassist-technology)
+[Intel QuickAssist Technology at `01.org`](https://01.org/intel-quickassist-technology)
 
 [Intel Device Plugin for Kubernetes](https://github.com/intel/intel-device-plugins-for-kubernetes)
 
@@ -68,7 +68,7 @@ needed to point to updated drivers or different install locations.
 
 ### Set environment variables (Every Reboot)
 
-Make sure to check [01.org](https://01.org/intel-quickassist-technology) for 
+Make sure to check [`01.org`](https://01.org/intel-quickassist-technology) for
 the latest driver.
 
 ```sh
@@ -135,7 +135,7 @@ $ sudo reboot
 
 ### Download Intel QAT drivers
 
-This will download the Intel QAT drivers from [01.org](https://01.org/intel-quickassist-technology). 
+This will download the Intel QAT drivers from [`01.org`](https://01.org/intel-quickassist-technology).
 Make sure to check the website for the latest version

```

```

Reply:
I added the quotes to those 3 instances and pushed another version. I also found that if I do the insecure module step before changing to the LTS kernel I don't have to do one of those reboots so I swapped the steps, updated the TOC, and removed one reboot reference.